documentation/decisions/adr/2023-10-16-email-marketing.md

```diff
@@ -0,0 +1,215 @@
+- **Deciders:** Lucas Brown, Aaron Couch, Billy Daly, Sarah Knopp, Sumi Thaiveettil
+- **Tags:** communucations, open source, email marketing
+
+## Context and Problem Statement
+
+An email marketing tool is primarily used for creating, sending, and tracking email campaigns to a list of subscribers. It allows for us to engage with our audience, push notifications and newsletters to our users, share information, and build customer relationships through email. 
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Usability:** Non-technical users should be able to access and create content with minimal training or guidance.
+- **Content Review:** Collaborators should be able to review and edit draft content before emails are sent out.
+- **Comments:** Reviewers should be able to leave in-line comments on content that they are reviewing.
+- **Internationalization (i18n):** The solution should provide support for displaying content in multiple languages.
+- **Analytics:** The platform should provide support for tracking open rates, click-through-rate and other other web analytics.
+- **Onboarding Costs:** Onboarding new members to the platform should be relatively inexpensive, both in terms of staff time/resources and direct costs (e.g. licensing fees).
+- **Maintenance Costs:** It should not be prohibitively expensive to maintain the email marketing tool, both in terms of staff time/resources and direct costs (e.g. hosting fees).
+- **Authority to Operate (ATO):** The tool should be FedRAMPed or have an Authority to Operate as it will have names and email addresses of public users. 
+
+#### Nice to Have
+
+- **Open Source:** The tool used to manage the email marketing content should be open source, if possible.
+- **Community & Support**: The platform has a strong community for ongoing support, updates, and bug fixes
+- **Scalability**: The platform should be able to hadndle a growing number of subscribers without performance issues
+
+## Options Considered
+
+
+- [Sendy](https://sendy.co/) 
+- [Salesforce Marketing](https://www.salesforce.com/products/engagement-marketing/) - 
+- [MailChimp](https://mailchimp.com/)
+- [Hubspot](https://www.hubspot.com/)
+- [Mautic](https://www.mautic.org/)
+- [Adobe Campaign](https://business.adobe.com/products/campaign/adobe-campaign.html)
+
+## Decision Outcome <!-- REQUIRED -->
+
+The suggested approach is to utilize Sendy and create a campaign for Simpler Grants.gov within the existing main account, leveraging existing knowledge and allowing for faster use. This allows for a faster initiation of the process. One of the most important criteria given by users of the tool, the Grants.gov Communications team, is the need to be able to send an email to the current 1M current subscribers quickly which Sendy cannot do. There are other limitations of this solution so we recommend that we select a back-up option email marketing solution for our long-term needs. 
```

Comment:
Can we make sure we create an ADR ticket for the long-term choice of tool?

Reply:
What do you think if we update this ADR with the long-term decision when that's decided? I can still create a ticket to update this one. I think we may not be able to be make a decision on the long-term tool until after our comms strategy kickoff or conversations with MicroHealth happen as it seems it would be nice to use the same tool for our all communications (NOFO, grants.gov, simpler grants.gov, etc.).